sigs.yaml

```diff
@@ -2648,7 +2648,9 @@ workinggroups:
   mission_statement: >
     Perform a security audit on k8s with a vendor and produce as artifacts a threat
     model and whitepaper outlining everything found during the audit.
-
+  charter_link:
```

Comment:
Tests should pass if you keep a newline here and remove the empty `charter_link` key

Reply:
Thanks Nikhita, I figured out that I need to run `make` to generate the files and not edit them all manually :')